### PR TITLE
Update local conda setup docs

### LINKED ISSUE #404: Update environment in conda setup docs
I think we probably do want to go back and reconsider the docs... I think we might have the docs follow the model here, but also show an option to install the packages into the `base` environment with (I think) `conda env update  -n base -f environment.yml`

_Originally posted by @jashapiro in https://github.com/AlexsLemonade/reproducible-research/pull/160#discussion_r1585450302_
            

---

We'll want to update the conda setup docs to have them create an `openscpca` environment as we do for the LSfR conda installation instrctions.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #404 

This PR updates conda setup docs so that local setup also creates an `openscpca` environment. I ended up making this into its own subsection to get all the details across in a way I thought was a bit more organized. Let me know what you think of the organization and of the content! I acknowledge that there are many admonitions, but since one is collapsed by default I think it's not too much?

### CODE REVIEW

**Review by jashapiro — COMMENTED** [review]
This looks good overall, but a couple order fixes, and a request for a bit more text...

**Comment by sjspielman** [comment]
Something that might be useful is to actually run the command and capture the _correct_ output from it! I've updated docs based on reviews, and the actual output from running the conda command.

File: docs/technical-setup/environment-setup/setup-conda.md [hunk 1]
```diff
@@ -55,30 +56,62 @@ These commands will set the [recommended channels](https://docs.conda.io/project
...
-    conda install awscli conda-lock jq pre-commit
+    conda env create -f environment.yml
     ```
-    <!-- Do we want to suggest this instead? `conda env update --name base --file environment.yml`? -->
 
-    - The [`awscli` package](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) will allow you to interact with [data stored in the Amazon Web Services (AWS) S3 bucket](../../software-platforms/aws/index.md)
-    - The [`conda-lock` package](https://conda.github.io/conda-lock/) is used to create fully reproducible, cross-platform [conda environments for analysis modules](../../contributing-to-analyses/determining-requirements/determining-software-requirements.md#managing-software-dependencies-with-conda)
-    - The [`jq` package](https://jqlang.github.io/jq/) provides JSON parsing capabilities
-    - The [`pre-commit` package](https://pre-commit.com) will allow you to use [pre-commit hooks when contributing to analysis modules](../../contributing-to-analyses/working-with-git/making-commits.md#pre-commit-checks)
+    When the process is complete, you'll see this message in the terminal providing the commands to activate and deactivate the environment:
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/setup-conda.md:80]:
```suggestion
    You will see a number of status messages flow by as packages are downloaded and installed.
    When the process is complete, you'll see the following message in the terminal, which includes the commands to activate and deactivate the environment:
```

File: docs/technical-setup/environment-setup/setup-conda.md [hunk 2]
```diff
@@ -55,30 +56,62 @@ These commands will set the [recommended channels](https://docs.conda.io/project
...
-1. At the end of the installation, you should see these messages in the terminal which indicate successful installation:
 
+
+1. As indicated in the message, activate the environment with the following command:
+
+    ```sh
+    conda activate openscpca
+    ```
+
+    At this point, your terminal prompt will have a prefix `(openscpca)`, which lets you know that you are in that conda environment.
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/setup-conda.md:108]:
```suggestion
    At this point, your terminal prompt will show the prefix `(openscpca)`, which lets you know that you are in that conda environment.
```

File: docs/technical-setup/environment-setup/setup-conda.md [hunk 3]
```diff
@@ -55,30 +56,62 @@ These commands will set the [recommended channels](https://docs.conda.io/project
...
+    In general, this is how conda indicates which environment, if any, the terminal is working in.
     <figure markdown="span">
-        ![Conda completed install output.](../../img/conda-success.png){width="275"}
+        ![Activate the openscpca environment](../../img/conda-activate-openscpca.png){width="300"}
     </figure>
 
-
-All set! 🎉
+    !!! tip "Activating the `openscpca` conda environment"
+        Whenever you are developing for OpenScPCA, you should work from this activated conda environment unless you are using a module-specific conda environment.
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/setup-conda.md:115]: We should probably link to the docs about module-specific envs here
```suggestion
        Whenever you are developing for OpenScPCA, you should work from either the `openscpa` conda environment or a module-specific conda environment.
```

**Review by jashapiro — COMMENTED** [review]
This looks good to me overall. Just a few relatively minor comments.

**Comment by sjspielman** [comment]
I added a new image with plain `bash` prompt, but even with this updated image, I don't know if we really need it at all with the text. Thoughts?

File: docs/technical-setup/environment-setup/setup-conda.md [hunk 4]
```diff
@@ -66,30 +67,67 @@ These commands will set the [recommended channels](https://docs.conda.io/project
...
+    ```sh
+    conda activate openscpca
+    ```
 
+    At this point, your terminal prompt will show the prefix `(openscpca)`, which lets you know that you are in that conda environment.
+    In general, this is how conda indicates which environment, if any, the terminal is working in.
     <figure markdown="span">
-        ![Conda completed install output.](../../img/conda-success.png){width="275"}
+        ![Activate the openscpca environment](../../img/conda-activate-openscpca.png){width="300"}
     </figure>
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/setup-conda.md:128]: Yeah, I have a feeling you don't need this. If you really wanted something, I might do the following chunk of code (basically reproducing your image) But I actually think the text you have is probably enough, especially given the potential variety in bash prompts.
````
```{ .console .no-copy title="Activated conda environment"}
bash$ conda activate openscpca
(openscpca) bash$ █
```
````

**Review by jashapiro — APPROVED** [review]
LGTM, and I will leave it up to you what to do with the active conda environment image or no image.

File: docs/technical-setup/environment-setup/setup-conda.md (hunk 4 above)
Comment (sjspielman) [reply docs/technical-setup/environment-setup/setup-conda.md:128]: > But I actually think the text you have is probably enough, especially given the potential variety in bash prompts.

**Pull request merged**